documentation/modules/metrics/archive_metrics-dashboard-descriptions.adoc

```diff
@@ -0,0 +1,211 @@
+
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics-config-files.adoc
+
+[id='ref-metrics-dashboards-{context}']
+= Example Grafana dashboards
+
+[role="_abstract"]
+If you deploy Prometheus to provide metrics,
+you can use Strimzi's example Grafana dashboards to monitor Strimzi components.
+
+Example dashboards are provided in the `examples/metrics/grafana-dashboards` directory as JSON files.
+
+All dashboards provide JVM metrics, as well as metrics specific to the component.
+For example, the Grafana dashboard for Strimzi operators provides information on the number of reconciliations or custom resources they are processing.
+
+The example dashboards don't show all the metrics supported by Kafka.
+The dashboards are populated with a representative set of metrics for monitoring.
```

Comment:
I wonder if we have some better word than representative. The dashboards have the metrics which we think are important for monitoring. But it is always opinionated, so other people might see other metrics as important. The word `representative` is ok, but at least for me it does not have exactly the same meaning.

Reply:
I changed to _The dashboards are populated with a limited set of metrics that are key for monitoring._ But we don't include this file now. I kept it in case we want to reintroduce.